frontend/src/scenes/authentication/signup/verify-email/VerifyEmail.tsx

```diff
@@ -44,26 +51,97 @@ export const VerifyEmailHelpLinks = (): JSX.Element => {
     )
 }
 
+export const VerifyEmailHelpLinks = (): JSX.Element => {
+    const [checkListValues, setCheckListValues] = useState<boolean[]>([])
+
+    const checklist = [
```

Comment:
I think this should be no more than 4 items - 3 actual suggestions and the funny one. Which one do we remove??